games-strategy/0ad_data/0ad_data-0.0.22~alpha.recipe

```diff
@@ -0,0 +1,28 @@
+SUMMARY="O A.D. - Game Data"
+DESCRIPTION="Data for the game 0 A.D."
+HOMEPAGE="https://play0ad.com/"
+COPYRIGHT="Wildfire Games"
+LICENSE="0ADLicense"
+REVISION="1"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-data.tar.xz"
```

Comment:
Thanks @BenjaminAmos for your update!

Oops, you forgot a tiny change in SOURCE_URI, probably because I had initially forgotten it myself in my previous review... 😢
```diff
-SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-data.tar.gz"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-${portVersion/\~/-}-unix-data.tar.xz"
```

I'll try to find some time to see if I can progress a little more before we merge the recipes (marked as untested.)